client/lib/checkout/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import { cartItems } from 'lib/cart-values';
+import { managePurchase } from 'me/purchases/paths';
+
+export function getExitCheckoutUrl( cart, siteSlug ) {
+	let url = '/plans/';
+
+	if ( cartItems.hasDomainRegistration( cart ) ) {
+		url = '/domains/add/';
```

Comment:
It would be nice to use some of the paths modules here, but I don't think we have any for these paths (yet).